tools/build_macos_dependencies.sh

```diff
@@ -1,5 +1,10 @@
 set -e -x
 
+# If USE_LEGACY_OPENGL is not set, default to "0"
+if [ -z "$USE_LEGACY_OPENGL" ]; then
+        USE_LEGACY_OPENGL="0"
+fi
```

Comment:
We could potentially use the shortcut for it:
```
USE_LEGACY_OPENGL="${USE_LEGACY_OPENGL:-0}"
```

Reply:
Suggestion applied! Thank you!